projects/project-player-library/src/lib/pages/files-page/files-page.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'lib-files-page',
+  // standalone: true,
```

Comment:
remove the comments if it isn't necessary

Reply:
done